src/actuator/ActuatorParsingSimple.C

```diff
@@ -0,0 +1,303 @@
+// Copyright 2017 National Technology & Engineering Solutions of Sandia, LLC
+// (NTESS), National Renewable Energy Laboratory, University of Texas Austin,
+// Northwest Research Associates. Under the terms of Contract DE-NA0003525
+// with NTESS, the U.S. Government retains certain rights in this software.
+//
+// This software is released under the BSD 3-clause license. See LICENSE file
+// for more details.
+//
+#include <actuator/ActuatorBulk.h>
+#include <actuator/ActuatorBulkSimple.h>
+#include <NaluParsing.h>
+#include <actuator/ActuatorParsingSimple.h>
+#include <NaluEnv.h>
+
+namespace sierra {
+namespace nalu {
+
+namespace {
+
+std::vector<double> 
+extend_double_vector(std::vector<double> vec, const int N)
```

Comment:
why not use `std::vector::resize`?

Reply:
See above, added some comments to `extend_double_vector()`.  Mostly used to validate the input and avoid ambiguous situations about how to resize the vector based on the inputs.